### DISCUSSION TITLE
Feature Request: Ability to see an author's entries quickly

### DISCUSSION THREAD

**Discussion by craftster** [craftcms/cms#8472] [Ideas]
We have several authors, and there are times when an editor needs to see the entries posted by a particular author. Clicking on the author name on the entry screen directs to the author profile, which does not list his/her entries.

Sorting the entries list by author isn't very useful either because it sorts it by alphabetical order and when you have thousands of entries with tens of authors, reaching a particular author using this method is very very time consuming. 

Last option that I figured would be putting the author name in the search field, but that doesn't yield accurate results. 

Ideally, clicking on the author name should always direct to a screen that lists all the entries of an author.

**Comment by craftster** [comment]
Even a solution like what Github has, would be very helpful ... <img width="570" alt="screenshot 2017-11-03 20 53 39" src="https://user-images.githubusercontent.com/33349037/32381614-20c45d94-c0d9-11e7-8e90-a2bcca102aca.png">

**Comment by brandonkelly** [comment]
Semi-related to #1630
